### PR TITLE
[REF] account_move_report: update version to make it consistent task#24545

### PR DESCRIPTION
**Pull request by keylor2906**

### CODE REVIEW

**Comment by keylor2906** [comment]
@luistorresm could you review/merge this

File: account_move_report/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 {
     "name": "Journal Entries Report",
-    "version": "saas~17.0.0.1",
+    "version": "11.0.0.0.1",
```
Comment (luistorresm) [inline account_move_report/__manifest__.py]: I think that bust be `11.0.1.0.0`

Is correct @moylop260 ?

Comment (moylop260) [reply account_move_report/__manifest__.py]: yep

Comment (moylop260) [reply account_move_report/__manifest__.py]: More info about:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

Comment (luistorresm) [reply account_move_report/__manifest__.py]: @keylor2906 

Could you make the improvements, please?

Regards

**Review by luistorresm — APPROVED** [review]
@moylop260 

Could you merge this PR, please?

Regards

**Pull request merged**